x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -48,6 +48,8 @@ export class BulkUploader {
 
     this._callClusterWithInternalUser = callClusterFactory(server).getCallClusterInternal();
 
+    const savedObjects = server.savedObjects;
+    this._savedObjectsClient = savedObjects.getScopedSavedObjectsClient(/* call with internal? */);
```

Comment:
This is in the background task on the server that calls collector fetch methods. It needs to be able to pass a saved objects client to the fetch methods, but this is outside of the context of a request.